physx.js

```diff
@@ -556,6 +556,35 @@ const physxWorker = (() => {
     return moduleInstance._makeTracker.apply(moduleInstance, arguments);
   }; */
   w.makePhysics = () => moduleInstance._makePhysics()
+  w.addJointPhysics = (physics, physicsId1, physicsId2, position1, position2, quaternion1, quaternion2, fixBody1 = false) => {
+    position1.toArray(scratchStack.f32, 0)
+    position2.toArray(scratchStack.f32, 3)
+    quaternion1.toArray(scratchStack.f32, 6)
+    quaternion2.toArray(scratchStack.f32, 10)
+
+    const position1Offset = scratchStack.f32.byteOffset
+    const position2Offset = scratchStack.f32.byteOffset + 3 * Float32Array.BYTES_PER_ELEMENT
+    const quaternion1Offset = scratchStack.f32.byteOffset + 6 * Float32Array.BYTES_PER_ELEMENT
+    const quaternion2Offset = scratchStack.f32.byteOffset + 10 * Float32Array.BYTES_PER_ELEMENT
+
+    const joint = moduleInstance._addJointPhysics(physics, physicsId1, physicsId2, position1Offset, position2Offset, quaternion1Offset, quaternion2Offset, fixBody1)

```

Comment:
Can we get rid of `fixBody`? It seems redundant, because we can simply perform the lock call right afterwards with the lock function.

Reply:
Deleted https://github.com/webaverse/app/pull/2731/commits/b72105e5452b9c47a53ee84df114d9fc1ddc7210